.github/workflows/CI.yml

```diff
@@ -60,6 +60,16 @@ jobs:
       run: |
         make test-dyno -j`util/buildRelease/chpl-make-cpu_count`
 
+  run_dyno_linters:
```

Comment:
In the future we might decide to merge this with `check_compiler_dyno_tests` since too many GH actions can actually end up running slower.